From 5994274082c0cf511f1f0c0852e2fe8a874ac0ad Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Wed, 28 Aug 2019 12:04:55 +0530
Subject: [PATCH 08/11] coresight: etm4x: Add Marvell OcteontTx CN9xxx ETM
 device

commit d51ad1e9deca6fde2b7bbf3fa7e48b9106ac4de4 from
git@git.assembla.com:cavium/WindRiver.linux.git

Add device id for Marvell OcteonTx CN9xxx ETM device.

Change-Id: If679a26626c5a2feb744af177cee438e3a1a26da
Signed-off-by: Linu Cherian <lcherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/16634
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/hwtracing/coresight/coresight-etm4x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/hwtracing/coresight/coresight-etm4x.c b/drivers/hwtracing/coresight/coresight-etm4x.c
index 2bce7cf0b0af..b60f7fbc78b4 100644
--- a/drivers/hwtracing/coresight/coresight-etm4x.c
+++ b/drivers/hwtracing/coresight/coresight-etm4x.c
@@ -1059,6 +1059,7 @@ static const struct amba_id etm4_ids[] = {
 	ETM4x_AMBA_ID(0x000bb95a),		/* Cortex-A72 */
 	ETM4x_AMBA_ID(0x000bb959),		/* Cortex-A73 */
 	ETM4x_AMBA_ID(0x000bb9da),		/* Cortex-A35 */
+	ETM4x_AMBA_ID(0x000cc210),		/* Marvell-OcteonTx-CN9xxx */
 	{},
 };
 
-- 
2.17.1

